src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
+    width: 40px;
+    object-fit: contain;
+    padding: 10px 0;
+  }
+
+  nav {
+    text-transform: uppercase;
+
+    .nav-list {
+      display: flex;
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      .nav-item {
+        transition: all ease-in-out 0.15s;
+        color: $nav-link-color;
+
+        &:not(:last-child) {
+          margin-right: 20px;
+        }
+
+        &:hover {
+          color: $hover-color;
+        }
+
+        .nav-link {
+          font-weight: 500;
+          font-size: 12px;
+          display: inline-block;
+          height: $header-height;
+          line-height: $header-height;
```

Comment:
Using the `$header-height` variable for `line-height` ensures consistency. However, ensure that this value aligns with the vertical rhythm and spacing rules.